src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -436,8 +436,10 @@ internal enum WellKnownMember
         System_ObsoleteAttribute__ctor,
 
         System_Span_T__ctor,
+        System_Span_T__ctorArray,
         System_Span_T__get_Item,
         System_Span_T__get_Length,
+        System_Span_T__op_Implicit_SpanReadOnlySpan,
```

Comment:
>System_Span_T__op_Implicit_SpanReadOnlySpan [](http://example.com/codeflow?start=8&length=43)

Is this still used?

Reply:
Yes, as we discussed offline, we'll use the well-known member for conversion.